### ISSUE TITLE
word list 

### ISSUE THREAD

**Issue by Steve502r** [HomelessPhD/Wealth_in_Poetry#2]
i actually scanned the article for all bip39 words but the list was way to big for me to manually try to guess. I cant write code but i’m gonna look for that list. i’ll get back to ya

**Comment by HomelessPhD** [comment]
My slow brute forcer goes with ~1500 mnemonics per second, i guess i have
not updated the github code yet

**Comment by GodFreyCoder** [comment]
I've seen ur code, but honestly trithemian and steganography not worked like that, puzzle owner might encrypt the words first and spread it into the words, for example from bip39 word i take "ability" and the owner encrypt it use trithemian cipher it will change into "hjrvtfl", he can spread it into a sentence and u will never found it. I guess coinmonks puzzle are made for no one to solve it. Only the owner know what rules he used to hide those passphrase.

**Comment by SmartArt09** [comment]
The text contains 448 total words out of the 2048 bip39 mnemonic words.
You can make your code a little better by just starting off with that many words.

I also tried brute-forcing for 12, 18 and 24 word mnemonics from the 448 words but all in vain.

**Comment by Steve502r** [comment]
If either one of you have all the words i have code that will do all combination possible with the private key, public key, and address along with it. Even checks the checksums are correct.  Im sure yall did this already but just in case. Sent from my iPhoneOn Jun 1, 2024, at 8:59 AM, SmartArt09 ***@***.***> wrote:﻿
The text contains 448 total words out of the 2048 bip39 mnemonic words.
You can make your code a little better by just starting off with that many words.
I also tried brute-forcing for 12, 18 and 24 word mnemonics from the 448 words but all in vain.

**Comment by Steve502r** [comment]
﻿If either one of you have all the words i have code that will do all combination possible with the private key, public key, and address along with it. Even checks the checksums are correct.  Im sure yall did this already but just in case. Sent from my iPhoneOn Jun 1, 2024, at 8:59 AM, SmartArt09 ***@***.***> wrote:﻿
The text contains 448 total words out of the 2048 bip39 mnemonic words.
You can make your code a little better by just starting off with that many words.
I also tried brute-forcing for 12, 18 and 24 word mnemonics from the 448 words but all in vain.

**Comment by HomelessPhD** [comment]
i also made a code to seek combinations of 12 words taken sequentially but
with different shifts, like 1,2,3,4,5,6,7,8,9,10,11,12;
1,3,4,5,6,7,8,9,10,11,12,13;
1,4,5,6,7,8,9,10,11,12,13,14;....,1,2,4,5,6,7,8,9,10,11,12,13;

but anyway i will not be able to check all

**Comment by SmartArt09** [comment]
Sorry, there's actually 264 words from the whole story that are valid BIP39 words.
I tried coding it in 2 different languages and realized I needed to do more data cleaning that I did earlier.

**Comment by Aarbi99** [comment]
what is the derivation path for the seeds ?

**Comment by SmartArt09** [comment]
> If either one of you have all the words i have code that will do all combination possible with the private key, public key, and address along with it. Even checks the checksums are correct.  Im sure yall did this already but just in case. Sent from my iPhoneOn Jun 1, 2024, at 8:59 AM, SmartArt09 ***@***.***> wrote:﻿
> The text contains 448 total words out of the 2048 bip39 mnemonic words.
> You can make your code a little better by just starting off with that many words.
> I also tried brute-forcing for 12, 18 and 24 word mnemonics from the 448 words but all in vain.
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Gotta do everything all over again because my laptop is now gone, so...no, I don't have all the words now.

But you can write a simple script yourself. Just remove all the punctuations in the article except the spaces and split the entire string, then see if any of the words in this list are there in the original 2048 BIP39 English wordlist. I mentioned the correct word count here in this thread in another reply.
Though again, going with all different possible combinations will take an eternity and then some, unless you have a quantum computer. :)

**Comment by Steve502r** [comment]
BIP 44Sent from my iPhoneOn Sep 9, 2024, at 9:31 PM, Aarbi99 ***@***.***> wrote:﻿
what is the derivation path for the seeds ?

**Comment by SmartArt09** [comment]
> BIP 44Sent from my iPhoneOn Sep 9, 2024, at 9:31 PM, Aarbi99 ***@***.***> wrote:﻿
> what is the derivation path for the seeds ?
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

The normal Bitcoin derivation path, at least that's what I used when I checked it.
If I remember correctly, it is m/44'/0'/0'/0 and then indexing starts from 0 (which is the most commonly used one) to however many addresses you want to generate, but just confirm it once from iancoleman.io/bip39

**Comment by SmartArt09** [comment]
Anyone found anything?

**Comment by HomelessPhD** [comment]
nothing so far as i know

**Comment by SmartArt09** [comment]
Damn, so all of the puzzles are really luck based...

**Comment by HomelessPhD** [comment]
Well... some of them i guess. Those that are seem easy - AR puzzles - are
also could be considered as lucky baseed. Cause some of us easily miss 1
correct words and some other would be lucky to catch that words
specifically.

**Comment by SmartArt09** [comment]
That's true. Any new story for this puzzle? Like somehow eliminating any of the total words?
I am so cooked rn that my Java program is performing worse than Python, at just 100 Mnemonics/Addresses per second _WITH THREADING_. 

There's 266 words in my code that are in the BIP39 English wordlist and following is the permutation/combination calculation for total population "n" = 266, if a sample "r" = 12, is chosen (most likely at random):

Permutations, nPr = 266! / (266 - 12)!            = 97,551,751,568,636,575,002,304,512,000
Combinations, nCr = 266! / 12! * (266 - 12)!  = 203,656,421,123,930,640,320

Doesn't seem so solvable now... :(
